css/properties/appearance.json

```diff
@@ -128,7 +134,7 @@
             ]
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
```

Comment:
I haven't reviewed a change to `experimental` before, but here's what I found:
- https://github.com/mdn/browser-compat-data/issues/1528
- https://developer.mozilla.org/en-US/docs/MDN/Contribute/Guidelines/Conventions_definitions#Experimental
- https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#status-information

With this shipping in two implementations now, it's no longer experimental 👍